src/wp-includes/load.php

```diff
@@ -164,8 +164,8 @@ function wp_check_php_mysql_versions() {
 
 	if ( ! function_exists( 'mysqli_connect' ) && ! function_exists( 'mysql_connect' )
 		// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.
-		&& ( defined( 'WP_CONTENT_DIR' ) && ! file_exists( WP_CONTENT_DIR . '/db.php' )
-			|| ! file_exists( ABSPATH . 'wp-content/db.php' ) )
+		&& ( ( ! defined( 'WP_CONTENT_DIR' ) || ! file_exists( WP_CONTENT_DIR . '/db.php' ) )
+			&& ! file_exists( ABSPATH . 'wp-content/db.php' ) )
```

Comment:
This change looks to be properly addressing the issue mentioned in the ticket description. 

issue mentioned in the description: _**This will check for the existence of db.php at the default path ABSPATH . 'wp-content/db.php', even if WP_CONTENT_DIR is defined and db.php is placed in WP_CONTENT_DIR.**_

But I think using a ternary would be more readable. Like this👇 :

`// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.`
`&& ! ( defined( 'WP_CONTENT_DIR' ) ? file_exists( WP_CONTENT_DIR . '/db.php' ) : file_exists( ABSPATH . 'wp-content/db.php' ) )`